integration-tests/tests/mocked/nino-check/nino-check-happy.test.ts

```diff
@@ -1,7 +1,7 @@
 import { HistoryEvent } from "@aws-sdk/client-sfn";
 import { SfnContainerHelper } from "./sfn-container-helper";
 
-jest.setTimeout(30_000);
```

Comment:
I'm assuming this delay is due to waiting for containers to startup.

Given a choice, I don't like increasing these timeouts, as it hides slow tests, and has a tendency to dramatically increase test runs when tests fail.

Is there any sort of healthcheck that can be monitored and used in https://jestjs.io/docs/configuration#globalsetup-string instead?

Reply:
I suppose we can create a ticket to investigate this https://govukverify.atlassian.net/browse/OJ-2188

Reply:
This is a good shoutout @merlinc but out of scope for this PR - agree we should haev a ticket to spike a solution and apply to all the relevant issues